source/sphinx_extensions/plugin_directory/templates/available.rst

```diff
@@ -7,5 +7,10 @@ QIIME 2 microbiome analysis functionality is made available to users via plugins
    :maxdepth: 3
 
    {% for name, _ in plugins|dictsort %}
+   {% if _.short_description %}
```

Comment:
Good thinking.  I think `name`, `plugin` is a good set of things to call the key-value pair, because then calling the parameters with `plugin.param_name` reflects the way you'd do it if you had the actual plugin object in Python so it's a pretty natural way to think about it, and it's clear that the `name` is the key, so that naming convention seems both lucid and nicely pythonic.  Anyway, thanks for pointing that out, and I've updated the code accordingly.